src/Controller/Component/PaginatorComponent.php

```diff
@@ -335,7 +336,10 @@ public function validateSort(RepositoryInterface $object, array $options)
             if (!in_array($direction, ['asc', 'desc'])) {
                 $direction = 'asc';
             }
-            $options['order'] = [$options['sort'] => $direction];
+
+            // Union the sort option from URL and default sort option to allow sorting by multiple fields
```

Comment:
If the sort conditions are always unioned, won't we lose the ability to sort on a single column?